generation/new_client/README.md

```diff
@@ -172,17 +172,25 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+This link must be found manually (until b/310975675 is implemented), and may not exist for every library, as they are generated adhoc.
+Poke around the product docs link to see if there is a valid REST API reference link.
 The value must starts with "https://".
 
-Example: `https://cloud.google.com/alloydb/docs`
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
+`--rest-docs="https://cloud.google.com/alloydb/docs/reference/rest" \`
 
 ### RPC Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+This link must be found manually (until b/310975675 is implemented), and may not exist for every library, as they are generated adhoc.
```

Comment:
Avoid starting a paragraph with "Note that..." sentence. Start with a sentence of how to get this value.

Reply:
Updated as there's now an official Cloud Drop field for it